### DISCUSSION TITLE
Sub-Statuses

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Lissy93/dashy#410] [Ideas]
A feature that would be useful for myself would be sub-statuses.
Here is a mockup:
![SubStatus](https://user-images.githubusercontent.com/20036947/148599885-c76fb83d-c2f5-4067-9a07-961ef61a8a6b.png)
I feel it would be handy for a few different scenarios, including when one has multiple of the service, or if a service has multiple pages that may need monitored.

**Comment by lissy93** [comment]
Thanks for the suggestion, nice idea. I will look into it, and maybe implement in the future :)
